grails-app/conf/application.yml

```diff
@@ -539,7 +539,8 @@ openboxes:
 
     # Allow users to customize logo image url as well as label
     logo:
-        url: "/assets/openboxes_logo_40x40.jpg"
+        # By default, the context path is /openboxes so lets use it here, because we cannot reference it in another way
+        url: "/openboxes/assets/openboxes_logo_40x40.jpg"
```

Comment:
Let's get rid of this change for now. If it actually needs to be fixed, let's do it in a separate PR against a ticket that focuses on this issue.

I assume that in places where logo.url is used, we can just deal with the context path ourselves in Grails. If we need the context path to be set for some reason, then we should do something like this for Spring Boot 1.3 (what Grails 3 uses)
```
server:
    context-path: /openboxes
```
... but we should also be able to override the context path at runtime using an environment variable 
```
export SERVER_CONTEXT_PATH=/openboxes
```
or command line argument
```
$ java -jar openboxes.war --server.context-path=/openboxes
```
But my hope was always that we'd deploy the application, the app name or context path would derived from the name of the war file and that would be passed into the application as-is. The only issue we were going to run into was that React didn't know the context path and that was going to need to be passed as a command line argument or environment variable.

If for some reasons I'm wrong, we should set this to be an external CDN URL? This would be a worst case scenario as it a little deceiving (we would essentially be adding one of those embedded images in emails that phones home to tell the sender who has opened their emails - I'm not a fan of those).